browser/brave_tab_helpers.cc

```diff
@@ -147,8 +153,14 @@ void AttachTabHelpers(content::WebContents* web_contents) {
 #endif
 
 #if BUILDFLAG(ENABLE_WIDEVINE)
+// We only need the DRM Tab Helper when Widevine is available through component
+// updates. This is the case on x64 Linux. On other architectures such as Arm64,
+// ENABLE_WIDEVINE is still set to let users supply Widevine themselves.
+// Only use the DRM Tab Helper when it is actually needed:
+#if !BUILDFLAG(IS_LINUX) || defined(ARCH_CPU_X86_64)
   BraveDrmTabHelper::CreateForWebContents(web_contents);
 #endif
+#endif  // BUILDFLAG(ENABLE_WIDEVINE)
```

Comment:
Instead of using `#ifdef`, how about using something like `BraveDrmTabHelper::MaybeCreateForWebContents()`?
and that static method could be no op if it's arm64 linux.